src/operation/watch/test.rs

```diff
@@ -0,0 +1,246 @@
+use std::time::Duration;
+
+use crate::{
+    bson::{doc, Document},
+    bson_util,
+    change_stream::ChangeStreamTarget,
+    cmap::{CommandResponse, StreamDescription},
+    operation::{Operation, Watch},
+    options::{
+        ChangeStreamOptions,
+        FullDocumentType,
+        ReadPreference,
+        SelectionCriteria,
+        StreamAddress,
+    },
+    test::TestClient,
+    Collection,
+    Namespace,
+};
+
+fn build_test(
+    target: impl Into<ChangeStreamTarget>,
+    pipeline: Vec<Document>,
+    options: Option<ChangeStreamOptions>,
+    mut expected_body: Document,
+) {
+    let target = target.into();
```

Comment:
`target` isn't used elsewhere, so it's fine to just pass `target.into()` directly as an argument to `Watch::new()` without making a separate binding

Reply:
fixed

Reply:
This doesn't seem to be fixed in the latest commit.

Reply:
didn't push earlier. fixed now